Dockerfile.QA

```diff
@@ -283,6 +283,18 @@ ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
 RUN GO111MODULE=off go get github.com/golang/protobuf/protoc-gen-go && \
     GO111MODULE=off go get google.golang.org/grpc
 
+# DCGM needed for GPU Metrics
```

Comment:
Can you double check if we need to install this in the QA container if we do it in our buildbase container?

Reply:
yep I checked. If I don't have it installed, there's tritonserver: error while loading shared libraries: libdcgm.so.2: cannot open shared object file: No such file or directory errors.

Reply:
This final stage of the QA container build is based on BASE_IMAGE which should be the production tritonserver image and that should have everything needed for tritonserver to run. So you shouldn't need to install libdcgm. The other installs you see here are installing things needed for the tests, not for tritonserver itself.

Reply:
@deadeyegoodwin would I need to do anything to make this happen? e.g. apply patches, or would all the installs from the `build` container automatically install everything

Reply:
The base container is not the build container, see my other comments in build.py. We must make sure that the QA container is just the production container (i.e. the base container as in master-py3-base) plus some installs needed for testing. Otherwise we risk testing tritonserver in a way that differs from how it is actually delivered.